### DISCUSSION TITLE
Override some translation's words - how to

### DISCUSSION THREAD

**Discussion by Nikoh77** [mastodon/mastodon#25095] [Q&A]
Hello, I am setting up my brand new mastodon instance and I would like to change just some words from the Italian translation while keeping compatibility for future updates, as is the case with the css.
Is it possible to use a custom translation file such as "it_alt.json" with at most 2 or 3 words which override and insert something like "include it.json"?

**Comment by Nikoh77** [comment]
No one helps me?....
Is there an irc channel or other way to have a little bit of support?
I posted here, I posted on mastodon but same result... no support

**Comment by renchap** [comment]
There is nothing built-in to support this. You will need to change the translations in `config/locales` and `app/javascript/locales/` directly.

If you feel that some things could be translated better, you can propose your translation changes in [CrowdIn](https://crowdin.com/project/mastodon).

**Reply by Nikoh77** [reply]
Hello and thanks for your reply, I just modified my language in `app/javascript/locales/`
e.g. I modified "reblog" in "boost", i restarted mastodon-web service but I keep seeing "reblog" on hover..... Why?

**Reply by renchap** [reply]
You need to recompile your Javascript files when you update anything in `app/javascript`: 

```RAILS_ENV=production rails assets:precompile```

**Reply by Nikoh77** [reply]
I recompiled but with another command: `RAILS_ENV=production bundle exec rails assets:precompile` is it wrong?

**Reply by renchap** [reply]
This seems correct

**Reply by Nikoh77** [reply]
so i don't understand, i'm confused, words changed, assets recompiled, services restarted but in the web interface i always see the same old words...

**Reply by renchap** [reply]
Which key did you change exactly in `app/javascript/mastodon/locales/it.json`?

**Reply by Nikoh77** [reply]
I want to change the on-over tooltip, for now "reblog" to "condividi"
